From: Angus Lees <gus@debian.org>
Date: Thu, 14 Jul 2022 13:17:39 +0200
Subject: Hardcode LLDB python module directory

Debian package installs python modules into a fixed directory, so
just hardcode path in wrapper script.

Forwarded: not-needed
---
 src/etc/rust-lldb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/src/etc/rust-lldb
+++ b/src/etc/rust-lldb
@@ -7,10 +7,10 @@
 host=$(rustc -vV | sed -n -e 's/^host: //p')
 
 # Find out where to look for the pretty printer Python module
-RUSTC_SYSROOT=$(rustc --print sysroot)
+RUSTC_SYSROOT="$(if type "$RUSTC" >/dev/null 2>&1; then "$RUSTC" --print=sysroot; else echo /usr; fi)"
 RUST_LLDB="$RUSTC_SYSROOT/lib/rustlib/$host/bin/lldb"
 
-lldb=lldb
+lldb=lldb-22
 if [ -f "$RUST_LLDB" ]; then
     lldb="$RUST_LLDB"
 else
